rep-2005.rst

```diff
@@ -0,0 +1,237 @@
+  * `osrf/osrf_pycommon <https://github.com/osrf/osrf_pycommon>`_
+  * `osrf/osrf_testing_tools_cpp <https://github.com/osrf/osrf_testing_tools_cpp>`_
+  * `ros/class_loader <https://github.com/ros/class_loader>`_
+  * `ros/pluginlib <https://github.com/ros/pluginlib>`_
+  * `ros2/eigen3_cmake_module <https://github.com/ros2/eigen3_cmake_module>`_
+  * `ros2/python_cmake_module <https://github.com/ros2/python_cmake_module>`_
+  * `ros2/rcutils <https://github.com/ros2/rcutils>`_
+  * `ros2/rcpputils <https://github.com/ros2/rcpputils>`_
+  * `ros2/ros_testing <https://github.com/ros2/ros_testing>`_
+
+* ROS interface pipeline
+
+  * `ros2/rosidl <https://github.com/ros2/rosidl>`_
+  * `ros2/rosidl_dds <https://github.com/ros2/rosidl_dds>`_
+  * `ros2/rosidl_defaults <https://github.com/ros2/rosidl_defaults>`_
+  * `ros2/rosidl_python <https://github.com/ros2/rosidl_python>`_
+  * `ros2/rosidl_runtime_py <https://github.com/ros2/rosidl_runtime_py>`_
+  * `ros2/rosidl_typesupport <https://github.com/ros2/rosidl_typesupport>`_
+
+* Interface definitions
+
+  * `ros-planning/navigation_msgs <https://github.com/ros-planning/navigation_msgs>`_
+  * `ros2/common_interfaces <https://github.com/ros2/common_interfaces>`_
+  * `ros2/example_interfaces <https://github.com/ros2/example_interfaces>`_
+  * `ros2/rcl_interfaces <https://github.com/ros2/rcl_interfaces>`_
+  * `ros2/test_interface_files <https://github.com/ros2/test_interface_files>`_
+  * `ros2/unique_identifier_msgs <https://github.com/ros2/unique_identifier_msgs>`_
+
+* RMW
+
+  * `ros2/rmw <https://github.com/ros2/rmw>`_
+  * `ros2/rmw_implementation <https://github.com/ros2/rmw_implementation>`_
+  * Connext (Connext itself is not open source)
+
+    * `ros2/rmw_connext <https://github.com/ros2/rmw_connext>`_
+    * `ros2/rosidl_typesupport_connext <https://github.com/ros2/rosidl_typesupport_connext>`_
+
+  * CycloneDDS
+
+    * `eclipse-cyclonedds/cyclonedds <https://github.com/eclipse-cyclonedds/cyclonedds>`_
+    * `ros2/rmw_cyclonedds <https://github.com/ros2/rmw_cyclonedds>`_
+
+  * FastRTPS
+
+    * `eProsima/Fast-CDR <https://github.com/eProsima/Fast-CDR>`_
+    * `eProsima/Fast-RTPS <https://github.com/eProsima/Fast-RTPS>`_
+    * `ros2/rmw_fastrtps <https://github.com/ros2/rmw_fastrtps>`_
+    * `ros2/rosidl_typesupport_fastrtps <https://github.com/ros2/rosidl_typesupport_fastrtps>`_
+
+  * OpenSplice
+
+    * `ros2/rmw_opensplice <https://github.com/ros2/rmw_opensplice>`_
+    * `ros2/rosidl_typesupport_opensplice <https://github.com/ros2/rosidl_typesupport_opensplice>`_
+
+* Client libraries
+
+  * `ros2/rcl <https://github.com/ros2/rcl>`_
+  * `ros2/rcl_logging <https://github.com/ros2/rcl_logging>`_
+  * `ros2/rclcpp <https://github.com/ros2/rclcpp>`_
+  * `ros2/rclpy <https://github.com/ros2/rclpy>`_
+
+* Orchestration
+
+  * `ros2/launch <https://github.com/ros2/launch>`_
+  * `ros2/launch_ros <https://github.com/ros2/launch_ros>`_
+
+* Features
```

Comment:
What is listed below are 
1. device drivers
2. algorithms
3. tools

Reply:
Could you propose a specific change here? Maybe through a PR?

Reply:
Seems like this has been resolved.